tests/test_queries.py

```diff
@@ -650,41 +672,43 @@ def make_query(self):
         )
         return self._model
 
-    def validate(self, expected_values=None):
-        validate_county_diurnal_hourly(
-            self.output_dir / self.name / "table.parquet", expected_values
+    def validate(self, expected_values):
+        filename = self.output_dir / self.name / "table.parquet"
+        df = read_parquet(str(filename))
+        assert not {"all_electricity_sum", "county", "hour"}.difference(df.columns)
+        hour = 16
+        val = (
+            df.filter("county == '06037'")
+            .filter(f"hour == {hour}")
+            .collect()[0]
+            .all_electricity_sum
         )
+        assert math.isclose(val, expected_values["la_electricity_hour_16"])
 
 
-class QueryTestDiurnalElectricityUseByCountyPrePostConcat(QueryTestBase):
```

Comment:
This test case is no longer valid because it relied on per-dataset aggregations.